### DISCUSSION TITLE
MacOS 14.5 Black flashing screen when presenting

### DISCUSSION THREAD

**Discussion by mayurankv** [jeertmans/manim-slides#470] [Q&A] [answered]
Hi, I'd like to use manim-slides for an academic presentation, but I would need to be able to reliably display the presentation from my laptop. I've seen the issues around the flashing black screen (as well as the pyqt bug report) and I saw that a few of these had people satisfied with the resolution through changing python package versions etc. However, I could not identify any clearly outlined set of versions which would mediate this issue. I was wondering whether any such set of versions exists? I have installed manim-slides in a lone virtual environment and the issue still persists so I'm a bit concerned this issue isn't addressable and I wouldn't be able to use this tool.

I'd really appreciate any direction or clarity. I know this has been addressed elsewhere so I'm sorry to waste your time, but I don't think it was made clear anywhere whether this was an unavoidable issue and whether it was platform agnostic (I seem to recall only seeing it referring to windows). Thank you!

**Comment by mayurankv** [comment]
Ah, I've reread the installation in a bit more detail and found that I have to use python3.11 instead of 3.12 to address this!

**Comment by jeertmans** [comment, marked as answer]
Hi @mayurankv, this is indeed a bug that is documented in both [the documentation](https://manim-slides.eertmans.be/v5.1.3/installation.html#optional-dependencies), and the issue #293. However, I recently merged a fix onto the main branch (see #465). It's not released on PyPI yet, but can you test locally that it fixes that black flashing screen on Python 3.12?

You can install the fix from the main branch with `pip install git+https://github.com/jeertmans/manim-slides.git@main`.

**Reply by mayurankv** [reply]
Hi! Would be happy to test this out. Have a deadline I'm trying to meet so will update you when I have a chance!

**Reply by jeertmans** [reply]
No issue, reach back to me when you had time to test :)
